modules/@apostrophecms/modal/ui/apos/mixins/AposDocsManagerMixin.js

```diff
@@ -146,6 +149,32 @@ export default {
     saveRelationship() {
       this.$emit('modal-result', this.checkedDocs);
       this.modal.showModal = false;
+    },
+    // Default implementation of isModified is based on whether the
+    // selection has changed, but you can override this and combine
+    // that bit with your own if your manager allows in-context editing
+    // of a piece (i.e. AposMediaManager)
+    isModified() {
```

Comment:
I don't see this actually used anywhere. The only manager using `isModified` is the media manager, which has its own implementation. Should this be checked in `saveRelationship`?

Reply:
```
    async confirmAndCancel() {
      let dismiss;
      if (this.isModified()) {
```

In AposModalModifiedMixin.

Reply:
True. I wasn't seeing them in concert in my head.